ush/templates/FV3LAM_wflow.xml

```diff
@@ -1494,6 +1498,47 @@ MODULES_RUN_TASK_FP script.
 
 {% endif -%}
 
+{%- if do_gsidiag_offline %}
+<!--
+************************************************************************
+************************************************************************
+-->
+  <task name="&ANAL_GSIDIAG_TN;_spinup" cycledefs="spinupcyc" maxtries="{{ maxtries_anal_gsi }}">
+
+    &RSRV_ANALYSIS;
+    &WALL_LIMIT_ANAL;
+
+    <command>&LOAD_MODULES_RUN_TASK_FP; "&RUN_ANAL_TN;" "&JOBSDIR;/JREGIONAL_RUN_GSIDIAG"</command>
+    &RESOURCES_GSIDIAG;
+    &NATIVE_ALL;
+    <walltime>&WALLTIME_GSIDIAG;</walltime>
+    &NODESIZE_ALL;
+    <jobname>&TAG;_&ANAL_GSIDIAG_TN;_spinup</jobname>
+    <join><cyclestr>&LOGDIR;/&ANAL_GSIDIAG_TN;spinup_&TAG;@Y@m@d@H.log</cyclestr></join>
+
+    <envar><name>GLOBAL_VAR_DEFNS_FP</name><value>&GLOBAL_VAR_DEFNS_FP;</value></envar>
+    <envar><name>PDY</name><value><cyclestr>@Y@m@d</cyclestr></value></envar>
+    <envar><name>cyc</name><value><cyclestr>@H</cyclestr></value></envar>
+    <envar><name>CDATE</name><value><cyclestr>@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>CYCLE_DIR</name><value><cyclestr>&CYCLE_BASEDIR;/@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>CYCLE_ROOT</name><value><cyclestr>&CYCLE_BASEDIR;</cyclestr></value></envar>
+    <envar><name>CYCLE_TYPE</name><value><cyclestr>spinup</cyclestr></value></envar>
+    <envar><name>GSI_TYPE</name><value><cyclestr>ANALYSIS</cyclestr></value></envar>
+    <envar><name>MEM_TYPE</name><value><cyclestr>MEMBER</cyclestr></value></envar>
+    <envar><name>SLASH_ENSMEM_SUBDIR</name><value><cyclestr></cyclestr></value></envar>
+    <envar><name>SATBIAS_DIR</name><value><cyclestr>&NWGES_BASEDIR;/satbias</cyclestr></value></envar>
+
+    <dependency>
+       <and>
+         <taskdep task="&ANAL_GSI_TN;_spinup"/>
+         <taskdep task="&HYBRID_RADAR_REF_TN;_spinup"/>
```

Comment:
Why do we need to depend on both ANAL_GSI_TN and HYBRID_RADAR_REF_TN? What about we don't do_envar_radar_ref?

Reply:
Added the dependence to check if we run HYBRID_RADAR_REF_TN or not.